ui/hooks/useCancelTransaction.js

```diff
@@ -34,47 +34,58 @@ export function useCancelTransaction(transactionGroup) {
   const conversionRate = useSelector(getConversionRate);
   const isMainnet = useSelector(getIsMainnet);
   const hideBasic = !(isMainnet || process.env.IN_TEST);
+
+  const [showCancelEditGasPopover, setShowCancelEditGasPopover] = useState(
+    false,
+  );
+
+  const closeCancelEditGasPopover = () => setShowCancelEditGasPopover(false);
+
   const cancelTransaction = useCallback(
     (event) => {
       event.stopPropagation();
-      if (isLegacyTransaction(primaryTransaction)) {
-        // To support the current process of cancelling or speeding up
-        // a transaction, we have to inform the custom gas state of the new
-        // gasPrice/gasLimit to start at.
-        dispatch(setCustomGasPriceForRetry(customGasSettings.gasPrice));
-        dispatch(setCustomGasLimit(GAS_LIMITS.SIMPLE));
-      }
-      const tx = {
-        ...primaryTransaction,
-        txParams: {
-          ...primaryTransaction.txParams,
-          gas: GAS_LIMITS.SIMPLE,
-          value: '0x0',
-        },
-      };
-      return dispatch(
-        showSidebar({
-          transitionName: 'sidebar-left',
-          type: 'customize-gas',
-          props: {
-            hideBasic,
-            transaction: tx,
-            onSubmit: (newGasSettings) => {
-              const userCustomizedGasTotal = getMaximumGasTotalInHexWei(
-                newGasSettings,
-              );
-              dispatch(
-                showModal({
-                  name: 'CANCEL_TRANSACTION',
-                  newGasFee: userCustomizedGasTotal,
-                  transactionId: primaryTransaction.id,
-                  customGasSettings: newGasSettings,
-                }),
-              );
-            },
+      if (process.env.SHOW_EIP_1559_UI) {
+        setShowCancelEditGasPopover(true);
+      } else {
+        if (isLegacyTransaction(primaryTransaction)) {
```

Comment:
this whole branch will go away. We'll do any necessary logic inside the EditGasPopover (setting initial gasPrice/maxFeePerGas/maxPriorityFeePerGas to be 10% higher)